youtube_dl/extractor/wikimedia.py

```diff
@@ -0,0 +1,69 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+import re
+
+from ..utils import (
+    clean_html,
+    determine_ext,
+    get_element_by_class,
+    urljoin,
+    compat_parse_qs,
+    ExtractorError)
+
+
+class WikimediaIE(InfoExtractor):
+    IE_NAME = 'wikimedia.org'
+    _NETRC_MACHINE = 'wikimediaorg'
+    _API_BASE_URL = 'https://commons.wikimedia.org/'
+    _VALID_URL = 'https://commons.wikimedia.org/wiki/File:(?P<id>[^/]+)'
+
+    _TEST = {
+        'url': 'https://commons.wikimedia.org/wiki/File:Die_Temperaturkurve_der_Erde_(ZDF,_Terra_X)_720p_HD_50FPS.webm',
+        'info_dict': {
+            'description': 'md5:7cd84f76e7081f1be033d0b155b4a460',
+            'ext': 'webm', 'id': 'Die_Temperaturkurve_der_Erde_(ZDF,_Terra_X)_720p_HD_50FPS',
+            'title': 'Die Temperaturkurve der Erde (ZDF, Terra X) 720p HD 50FPS.webm - Wikimedia Commons',
+            'license': 'md5:62907cddf705a9f7ae7076c15407a977',
+            'author': None, 'subtitles': {'de': [{'ext': 'srt',
+                                                  'url': 'https://commons.wikimedia.org/w/api.php?action=timedtext&amp;title=File%3ADie_Temperaturkurve_der_Erde_%28ZDF%2C_Terra_X%29_720p_HD_50FPS.webm&amp;lang=de&amp;trackformat=vtt '}],
+                                          'en-gb': [{'ext': 'srt',
+                                                     'url': 'https://commons.wikimedia.org/w/api.php?action=timedtext&amp;title=File%3ADie_Temperaturkurve_der_Erde_%28ZDF%2C_Terra_X%29_720p_HD_50FPS.webm&amp;lang=en-gb&amp;trackformat=vtt '}],
+                                          'nl': [{'ext': 'srt',
+                                                  'url': 'https://commons.wikimedia.org/w/api.php?action=timedtext&amp;title=File%3ADie_Temperaturkurve_der_Erde_%28ZDF%2C_Terra_X%29_720p_HD_50FPS.webm&amp;lang=nl&amp;trackformat=vtt '}],
+                                          'en': [{'ext': 'srt',
+                                                  'url': 'https://commons.wikimedia.org/w/api.php?action=timedtext&amp;title=File%3ADie_Temperaturkurve_der_Erde_%28ZDF%2C_Terra_X%29_720p_HD_50FPS_-_redub_NL.webm&amp;lang=en&amp;trackformat=vtt '}]}}
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        ext = determine_ext(url, None)
+        if ext is None:
+            raise ExtractorError('invalid video url', expected=True)
+        webpage = self._download_webpage(url, video_id)
+        self.report_extraction(video_id)
+        video_url = self._html_search_regex('<source [^>]*src="([^"]+)"', webpage,
+                                            'video URL')
+        license = get_element_by_class('layouttemplate licensetpl mw-content-ltr', webpage)
+        license = clean_html(license)
+
+        description = get_element_by_class('description', webpage)
+        author = self._html_search_regex('<td\b[Author>]*>[\n\s]([^<]+?)\s*</td>', webpage, 'video author',
```

Comment:
Fixing https://github.com/ytdl-org/youtube-dl/pull/30796#issuecomment-1124474245. This will show up in the Linter CI test that I've now enabled. Although the main problem is plain quoting instead of raw (`r'xxx'`), the pattern wouldn't have worked anyway.


```suggestion
        author = self._html_search_regex('>\s*Author\s*</td>\s*<td\b[^>]*>\s*([^<]+?)\s*</td>',
                                         webpage, 'video author', default=None)
```